src/learn/view/learnpagemodel.cpp

```diff
@@ -134,9 +134,13 @@ QVariantList LearnPageModel::playlistToVariantList(const Playlist& playlist) con
     auto formatDuration = [](int durationSecs) {
         int seconds = durationSecs;
         int minutes = seconds / 60;
+        seconds -= minutes * 60;
         int hours = minutes / 60;
+        minutes -= hours * 60;
 
-        return (hours > 0 ? QString::number(hours) + ":" : "") + QString::number(minutes % 60) + ":" + QString::number(seconds % 60);
+        return (hours > 0 ? QString::number(hours) + ":" : "")
+               + ((minutes < 10) ? "0" : "") + QString::number(minutes) + ":"
```

Comment:
Or you can use QString::rightJustified like 
QString::number(seconds % 60).rightJustified(2, '0')

Reply:
Wasn't aware of `rightJustified` but sure, that could be used as well.

Do we want this for minutes if hours == 0 too, or not?

Reply:
Doesn't make sense for hours though imho, as there is never anything preceding it.
As such I'd only right justify if there is something before it (so always for seconds and for minutes only if there are hours). This would follow the model of many existing video sites (YouTube, Vimeo, ...)

Perhaps @bkunda has an opinion here?

Reply:
@jeetee yes, it makes sense with hours (like in YouTube).
Please fix this and we will merge your PR.

Reply:
updated using rightJustified and following YouTube format logic in [d96e05e](https://github.com/musescore/MuseScore/pull/13992/commits/d96e05e0bb55abc5c2c534862c03cd1b2880f818)